collectors/python.d.plugin/python.d.conf

```diff
@@ -88,6 +88,7 @@ nginx_log: no
 # redis: yes
 # rethinkdbs: yes
 # retroshare: yes
+# riak: yes
```

Comment:
riak_kv

Reply:
Missed that! Fixed in fe218783995f039712fc4feca8b446a331c64762.